specfile/spec_parser.py

```diff
@@ -99,6 +99,20 @@ def _make_dummy_sources(
         Yields:
             List of paths to each created dummy source.
         """
+
+        def write_magic(path, magic):
```

Comment:
docstrings with a little bit of explanation could be helpful here

Reply:
I've added a comment.

Reply:
thanks!